Cargo.toml

```diff
@@ -38,9 +38,9 @@ members = [
   "pallets/anchors",
   "pallets/bridge",
   "pallets/block-rewards",
-  "pallets/connectors",
-  "pallets/connectors-gateway",
-  "pallets/connectors-gateway/routers",
+  "pallets/liquidity-pools",
+  "pallets/liquidity-pools-gateway",
+  "pallets/liquidity-pools-gateway/routers",
```

Comment:
Could you please apply lexicographical ordering? Thanks!

Reply:
I am not sure how you would like this order to be different? Or do you mean your comment for the whole file?

Reply:
I was referring to moving down `pallets/liquidity-pools-*` some lines down as the next pallet is `claims` which is lexicographically smaller.

Reply:
Done